packages/lib/ProtocolUtils.ts

```diff
@@ -0,0 +1,37 @@
+const URL = require('url-parse');
```

Comment:
ProtocolUtils should be lowercase "protocolUtils". In general, util files should be lowercase and classes uppercase.

But in fact could you call it "callbackUrlUtils"?